docs/includes/startup_error_behavior.md

```diff
@@ -9,3 +9,6 @@ using the `startup_error_behavior` setting. Available values are:
 - `retry`:  Telegraf will try to startup the plugin in every gather or write
             cycle in case of startup errors. The plugin is disabled until
             the startup succeeds.
+- `probe`:  Telegraf will call the `Probe() error` method, if available. If the
+            method returns an error, Telegraf disables the plugin but continues
+            processing for all other plugins.
```

Comment:
I would make the message a bit less technical and add information on what happens if the plugin does **not** support probing...
```suggestion
- `probe`:  Telegraf will probe the plugin's function (if possible) and disabled the plugin 
            in case probing fails. If the plugin does not support probing Telegraf will...
```